libs/membership/website/src/lib/subscribe/subscribe-container.tsx

```diff
@@ -61,6 +63,12 @@ export const SubscribeContainer = <T extends Exclude<BuilderUserFormFields, 'fla
     }
   })
 
+  const [resubscribe] = useResubscribeMutation({
+    onCompleted() {
+      window.location.href = '/thank-you'
```

Comment:
Shouldn't the payment provider redirect us? It's set what page in the memberPlan